source/connect/stable-api.txt

```diff
@@ -0,0 +1,119 @@
+.. _cpp-stable-api:
+
+==============
+{+stable-api+}
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: compatible, backwards, upgrade
+
+.. note::
+
+   The {+stable-api+} feature requires {+mdb-server+} 5.0 or later.
+
+Overview
+--------
+
+In this guide, you can learn how to specify **{+stable-api+}** compatibility when 
+connecting to a MongoDB deployment.
+
+The {+stable-api+} feature forces the server to run operations with behaviors compatible 
+with the API version you specify. When you update either your driver or server, 
+the API version changes, which can change the way these operations behave.
+Using the {+stable-api+} ensures consistent responses from the server and 
+provides long-term API stability for your application.
+
+The following sections describe how you can enable and customize {+stable-api+} for
+your MongoDB client. For more information about the {+stable-api+}, including a list of 
+the commands it supports, see :manual:`Stable API </reference/stable-api/>` in the
+{+mdb-server+} manual.
+
+Enable the {+stable-api+}
+-------------------------
+
+To enable the {+stable-api+}, perform the following steps:
+
+1. Construct a ``mongocxx::options::server_api`` object and specify a {+stable-api+}
+   version. You must use a {+stable-api+} version defined in the
+   ``mongocxx::options::server_api::version`` enum. Currently, the driver supports only
+   version 1 (``k_version_1``).
+#. Construct a ``mongocxx::options::client`` object. Set the ``server_api_opts`` field
+   of this object to the ``server_api`` object you created in the previous step.
+#. Construct a ``mongocxx::client`` object, passing in your ``mongocxx::uri`` object
+   and the ``mongocxx::options::client`` object you created in the previous step.
```

Comment:
This is more of a general comment, but the length of the objects referenced here (i.e. `mongocxx::options::server_api`) makes this a bit unwieldy to read. If this is the agreed-upon way of referencing these then there's nothing to be done, but felt it was worth mentioning.

Reply:
totally get it. yeah, this is the idiomatic way to refer to these types.